build/board.sh

```diff
@@ -10,7 +10,9 @@ XRTBUILD=$(readlink -f $(dirname ${BASH_SOURCE[0]}))
 # Environment
 ################################################################
 xrt=$XRTBUILD/Release/opt/xilinx/xrt
-
+if [ x"${XRT_INSTALL_PREFIX}" != "x" ]; then
+    xrt="$XRTBUILD/Release"$XRT_INSTALL_PREFIX"/xrt"
+fi
```

Comment:
Where does XRT_INSTALL_PREFIX come from?  I don't see it is this script?  
board.sh is a very old legacy script, I don't think it needs to support anything but the default install prefix, I say we revert the changes in this one file.

Reply:
Ok. I will remove it.